src/html/why.html

```diff
@@ -0,0 +1,52 @@
+@@include("partials/head.html", {"tagline": false})
+
+    <main>
+        <div class="container pt3 pb3">
+
+            <h2>Why Meadowrun?</h2>
+
+            <p>The short answer is: Meadowrun runs your Python code in the cloud, seamlessly. Scale up to a bigger machine, or scale out to many machines – once you are ready to run your analysis or program on a dataset that doesn’t fit on your laptop, Meadowrun helps you do that without hassle and while keeping many of the benefits of running locally.</p>
```

Comment:
~The short answer is~: Meadowrun runs your Python code in the cloud, seamlessly. Scale up to a bigger machine ~,~ or scale out to many machines – once you are ready to run your analysis or program on a dataset that doesn’t fit on your laptop, Meadowrun helps you do that without hassle and while keeping many of the benefits of running locally.